src/main/java/uk/gov/hmcts/sptribs/caseworker/event/page/RecordNotifyParties.java

```diff
@@ -47,6 +49,10 @@ public AboutToStartOrSubmitResponse<CaseData, State> midEvent(CaseDetails<CaseDa
 
         if (checkNullSubjectRepresentativeRespondent(data)) {
             errors.add("One recipient must be selected.");
+        } else if ((data.getCicCase().getCaseSubcategory() == CaseSubcategory.FATAL
+            || data.getCicCase().getCaseSubcategory() == CaseSubcategory.MINOR)
+            && !CollectionUtils.isEmpty(data.getCicCase().getNotifyPartySubject())) {
+            errors.add("Subject should not be selected for notification if the case is Fatal or Minor");
```

Comment:
Do we have some sort of Constants class where we can declare literals like this, as public static etc.?
As it's used in the class below, too.

Reply:
we have EventContants but it is mostly for event names and special characters.
I can move to there though, or create a new one. Up to you ?